apps/yarn.lock

```diff
@@ -2225,6 +2227,11 @@
     "@svgr/plugin-svgo" "^4.0.3"
     loader-utils "^1.1.0"
 
+"@types/estree@*":
```

Comment:
I'm trying to remember whether it was typical that we would end up introducing new dependencies in the main repo when bumping a version of our repo, when we were doing `ml-activities` for Oceans.  @madelynkasula or @bethanyaconnor do you remember?

Reply:
Oh, I found [an example](https://github.com/code-dot-org/code-dot-org/pull/32362/files) where we did introduce new dependencies, so I guess it's fine!